lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -3,6 +3,8 @@ defmodule ExUnit.Runner do
 
   alias ExUnit.EventManager, as: EM
 
+  @rand_algorithm :exsplus
```

Comment:
In stream_data we use `:exs1024` as the algorithm, but as far as I understand in OTP 20 all previous algorithms are soft deprecated and replaced by three new algorithms. We can either

1. go with the old ones, but then we should sync between stream_data and Elixir or
1. do something like `:rand.seed_s(:exs1024s)` here and catch `FunctionClauseError`s to fall back to a deprecated algorithm but I don't know if that would cause weird interactions

Thoughts @josevalim?

Reply:
We can pick whatever choice you want as long as we are compatible with OTP 19 and OTP 20. If we cannot use exs1024 with OTP 19 then we should probably continue with exsplus and add a task to #5851.

Reply:
Does it make sense to allow the user to configure the algorithm through `ExUnit.configure`?

Reply:
If someone needs it, they can ask for it. :) But I can't think of any reason upfront.